package.json

```diff
@@ -2,10 +2,8 @@
   "name": "eslint-config-crowdstrike",
   "version": "11.0.1",
   "description": "ESLint config for CrowdStrike",
-  "main": "index.js",
```

Comment:
changing this package to ` module`. Would definitely need to be released as a major version as this is a breaking change